src/palette_colors_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/alecthomas/assert"
+)
+
+// FIXME: test is not completely deterministic, engineReder calls can have different
+// outputs due to time - there can be 1 second difference on the clocks during
+// rendering. The test is useful, and removing it completely will be incorrect - having
+// a close-to-real test of no palette vs palette rendering parity is better than not
+// having one. Not sure what to do with this one.
+func TestEngineRendersSamePrompt(t *testing.T) {
```

Comment:
Also, @JanDeDobbeleer, seeking some advice about this test. Do you have any ideas on how to make it correct?

Reply:
I don't see why this is necessary, if it renders the colors as we expect, no need to compare with or without. Tghi sis an ee2e test, for which I don't see the added benefit if your low level functions are properly unit tested.

Reply:
Makes sense, I will remove it then.

Reply:
Removed.